### ISSUE TITLE
Storyboarder really slow: Storyboarder Helper issue?

### ISSUE THREAD

**Issue by raffmarqs** [wonderunit/storyboarder#1373]
Hey guys, I've been using Storyboarder lately but it's running too slow and slowing the whole computer as well. I don't know if it's a matter of computer performance or an issue with multiple instances of storyboarder helper that are appearing in my Activity Monitor.

**"Issue"**
![image](https://user-images.githubusercontent.com/1461653/45882200-c481b380-bd83-11e8-880e-606250c0a581.png)

**My config**
![image](https://user-images.githubusercontent.com/1461653/45882265-06aaf500-bd84-11e8-8140-b18578e293b6.png)

**Storyboarder Version:** 1.8.0

Let me know if you need any other information or if you know how to solve this. Thanks!

**Comment by audionerd** [comment]
Have you tried turning off the “High Quality Drawing Engine”? https://wonderunit.com/storyboarder/faq/#Drawing-is-laggy-or-slow-How-can-I-make-it-faster

**Comment by raffmarqs** [comment]
Yes, it's off. The problem isn't the drawing engine because I'm not drawing anything. Just opening the app and using it to reorder shots, fill the voice lines and other non-drawing activities makes everything slow. I also have a friend experiencing the same issue.

**Comment by ksalihcavusoglu** [comment]
Same for me. 16 GBRAM and 8 cores.
